cypress/integration/form-builder/conditionalFieldset.test.js

```diff
@@ -37,6 +37,12 @@ const doc = {
   singleReadOnlyCallbackTrue1: 'Lorem',
 }
 
+const waitForReviewChanges = () => {
+  // Sometimes it takes a while for Review Changes to appear when loading the page.
+  // This prevents the test from failing randomly
+  cy.contains('Changes', {timeout: 10000}).should('be.visible')
```

Comment:
What exactly does `Changes` represent? Can we not check if the element with the review changes test id is visible instead? `Changes` feels a little vague

Reply:
Added testid to the review changes panel instead.